InfinityWorks.TechTest.Test/Data/FsaClientTestData.cs

```diff
@@ -0,0 +1,64 @@
+﻿using InfinityWorks.TechTest.Model;
+using InfinityWorks.TechTest.Test.Resources;
+using Newtonsoft.Json;
+using System.Collections.Generic;
+
+namespace InfinityWorks.TechTest.Test.Data
+{
+    internal static class FsaClientTestData
+    {
+        public static string AuthorityData => ResourceRetriever.ReadResource("Authorities").Result;
+        public static string Establishments => ResourceRetriever.ReadResource("Establishments").Result;
+
+        public static FsaAuthorityList GetAuthorityData()
+        {
+            return JsonConvert.DeserializeObject<FsaAuthorityList>(AuthorityData);
+        }
+
+        public static IEnumerable<AuthorityRatingItem> GetExpectedRatingItemData()
+        {
+            return new List<AuthorityRatingItem>
+            {
+                new AuthorityRatingItem
+                {
+                    Name = "5",
+                    Value = 55
```

Comment:
These values are all rounded, but we tell candidates _not_ to round the results (it's done in Javascript).
Because it's a floating point value, the test assertion will need to factor in a margin of error.